### PR TITLE
feat: new demo app

### PR DESCRIPTION
**Pull request by JesseKuntz**
Adding the new demo app:

https://github.com/user-attachments/assets/08813e47-cc18-49bc-8bae-33086a88754c

### CODE REVIEW

**Comment by guillaumegaluz** [comment]
Mike - i changed the bg color because white in the central panel didnt work as it doesnt have enought contrast with the widget. make tweaks, as long as we keep enough contrast

File: demo/src/components/MainContent.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ export function MainContent({
                 UI components for embeddable staking
               </p>
             </div>
-            <div className="my-8 mx-auto w-[350px] h-[393px] overflow-hidden border border-[#ddd] rounded-md">
+            <div className="my-8 mx-auto w-[350px] h-[450px] overflow-hidden">
```
Comment (JesseKuntz) [inline demo/src/components/MainContent.tsx:131]: @guillaumegaluz this is a problematic change - the default wallet connection on ETH needs at least a height of 450px

File: demo/src/components/ToggleGroup.tsx [hunk 2]
```diff
@@ -46,24 +85,37 @@ export function ToggleGroup<T extends string>({
...
+                </span>
+              )}
+              <span>{item.label}</span>
             </button>
             {showTooltip && item.disabled && (
               <div className="absolute opacity-0 group-hover:opacity-100 transition-opacity duration-200 bottom-full left-1/2 -translate-x-1/2 mb-2 px-2 py-1 bg-gray-900 text-white text-xs rounded pointer-events-none whitespace-nowrap font-inter">
                 {item.value === "mainnet"
                   ? "Demo uses testnets only"
-                  : item.value === "custom"
-                  ? "Custom wallets coming soon"
```
Comment (JesseKuntz) [inline demo/src/components/ToggleGroup.tsx]: @guillaumegaluz did you intentionally remove this? I think for now we should keep until we fix custom wallets.

Comment (guillaumegaluz) [reply demo/src/components/ToggleGroup.tsx]: not intentional. adding it back

File: demo/src/components/MainContent.tsx (hunk 1 above)
Comment (JesseKuntz) [reply demo/src/components/MainContent.tsx:131]: To add, this is obviously not ideal - we just haven't yet figured out "auto-resizing" of the iframe.

**Review by ybce — APPROVED** [review]

**Pull request merged**